src/App.tsx

```diff
@@ -1,9 +1,67 @@
 import './App.scss';
+import {
+  Routes, Route, NavLink, Navigate,
+} from 'react-router-dom';
+import { PeoplePage } from './components/PeoplePage';
+import { NotFound } from './components/NotFound';
 
-const App = () => (
-  <div className="App">
-    <h1>People table</h1>
-  </div>
-);
+const App = () => {
+  return (
+    <div className="App">
+      <div className="tabs is-centered">
+        <ul>
+          <li>
+            <NavLink
+              to="/"
+              style={({ isActive }) => {
+                return {
+                  color: isActive ? 'blue' : '',
+                };
+              }}
+            >
+              Home Page
+
+            </NavLink>
+          </li>
+          <li>
+            <NavLink
+              to="/PeoplePage"
+              style={({ isActive }) => {
+                return {
+                  color: isActive ? 'blue' : '',
+                };
+              }}
+            >
+              People Page
+
+            </NavLink>
+          </li>
+        </ul>
+      </div>
+
+      <Routes>
+        <Route
+          path="*"
+          element={<NotFound />}
+        />
+        <Route
+          path="/"
+          element={<p className="title">Home Page</p>}
+        />
+
+        <Route
+          path="/home"
+          element={<Navigate to="/" />}
```

Comment:
don't forget to add `replace`
here is why – https://reach.tech/router/api/navigate (at the bottom of the article)